providers/curator.rb

```diff
@@ -18,6 +18,7 @@ def load_current_resource
   @log_file = new_resource.log_file || Logstash.get_attribute_or_default(node, @instance, 'curator_cron_log_file')
   @user = new_resource.user || Logstash.get_attribute_or_default(node, @instance, 'user')
   @bin_dir = new_resource.user || Logstash.get_attribute_or_default(node, @instance, 'curator_bin_dir')
+  @index_prefix = new_resource.user || Logstash.get_attribute_or_default(node, @instance, 'curator_index_prefix')
```

Comment:
I think an error on line 20 lead to an error on line 21 as well...

Why does @bin_dir get set to new_resource.user if set?
And so, why does @index_prefix get set to resource.user if set?

I would have expected to see this:

```ruby
# providers/curator.rb (lines 19-21)
@user = new_resource.user || Logstash.get_attribute_or_default(node, @instance, 'user')
@bin_dir = new_resource. bin_dir || Logstash.get_attribute_or_default(node, @instance, 'curator_bin_dir')
@index_prefix = new_resource.index_prefix || Logstash.get_attribute_or_default(node, @instance, 'curator_index_prefix')

# resources/curator.rb
attribute :bin_dir,           kind_of: String
attribute : index_prefix, kind_of: String
```